DB/seedDB.js

```diff
@@ -0,0 +1,76 @@
+const faker = require('faker');
+const fs = require('fs');
+
+(function generateData(number, batch) {
+  console.time('timer');
+  const clothes = ['hoodies', 'shirts', 'dress', 'shoes', 'pants', 'jacket'];
+  const randClothesIndex = Math.floor(Math.random() * clothes.length);
+  const keyword = clothes[randClothesIndex];
+  const pictureLength = Math.ceil(Math.random() * 7);

```

Comment:
cool that you added this bit of complexity with the randomly-generated picture length to challenge yourself